draft-ietf-tls-esni.md

```diff
@@ -882,6 +882,13 @@ desired.
 Moreover, as noted in the introduction, SNI encryption is less useful
 without encryption of DNS queries in transit via DoH or DPRIVE mechanisms.
 
+## Related Privacy Leaks
+
+ESNI requires encrypted DNS to be an effective privacy protection mechanism.
+Similarly, implementations may wish to prohibit cleartext OCSP traffic in the
```

Comment:
The PR looks good to me.

That said, it might be to a good idea to say something like "prohibit generating OCSP traffic, especially in cleartext", as using an encrypted channel to the OCSP responder could still be an issue in sense that it leaks the IP address of the responder (or because use of OCSP is an act of leaking information to the OCSP responder).

Reply:
OCSP or CRL.

I think that this needs to be framed differently.  The point here is that OCSP (or CRL checks) can leak the name.  But the reaction I would encourage is OCSP stapling, not "don't do OCSP" as this would appear to advocate for.

Reply:
Perhaps requiring stapling with a SHOULD is fine? (It seems fine to me, at least.)